test/test-remote/looker/src/args.ts

```diff
@@ -361,7 +400,7 @@ export function parseCmdlineArgs(): void {
     }
   }
 
-  if (CFG.max_concurrent_writes === 0) {
+  if (CFG.max_concurrent_writes === 0) { // TODO this is always zero - remove from CFG?
```

Comment:
> this is always zero - remove from CFG?

I don't understand that part :-)

This is what was set via `--max-concurrent-writes <N>` and I've found this to be one of the most important parameters, I pretty much always set it.

Reply:
Oh this is my fault, I had written this comment early on and then forgot about it. I had done a text search for `max_concurrent_writes` and didn't see any setters (hadn't realized at the time that I should've also checked `max-concurrent-writes`)